pgmgr/pgmgr_test.go

```diff
@@ -367,6 +369,34 @@ func TestMigrateCustomMigrationTable(t *testing.T) {
 	}
 }
 
+func TestMigratePsqlDriver(t *testing.T) {
```

Comment:
Is there a test for a migration file with invalid syntax?  If so, the migration should bail and return exit code 1.  With `psql` at least, this means adding the command line argument `-v "ON_ERROR_STOP=1"`.

Reply:
Good call; that case wasn't working. I added that flag, along with `-1`, and things look good.